include/libweston/backend-rdp.h

```diff
@@ -33,21 +33,19 @@ extern "C" {
 #include <libweston/libweston.h>
 #include <libweston/plugin-registry.h>
 
+#include <freerdp/freerdp.h>
```

Comment:
This is a bit concern as anyone includes this header needs to have dependencies to FreeRDP, like rdprail-shell in this PR. Should we have our own rdp monitor struct rather than using same from FreeRDP?

Reply:
I think it's OK to require this dependency for anyone that needs to include the backend-rdp header. If they're including our backend-rdp.h header, there's some expectation that they may have to deal with FreeRDP specifics.

Reply:
Yeah. It would be great if FreeRDP had a more stable API, but given that frontends are going to have to deal with it anyway, I don't think there's any value in defining our own wrapper struct. After all, if libweston can create an rdpMonitor equivalent which expresses everything required & is ABI-stable - why couldn't that struct just be provided by FreeRDP itself?

Reply:
> I think it's OK to require this dependency for anyone that needs to include the backend-rdp header. If they're including our backend-rdp.h header, there's some expectation that they may have to deal with FreeRDP specifics.

But that's not true for rdprail-shell, they don't access any FreeRDP API or struct, but now because it's including backend-rdp header, it has to take dependency to FreeRDP.

Reply:
There are definitely some very simple shells who wouldn't require it for sure. But when you get into providing more involved functionality like audio, clipboard, app IDs, etc, then the frontend is required (after some of these PRs) to deal with RDP directly.

Reply:
OK, then keep as-is, thanks for the response.